src/common/pool_map.c

```diff
@@ -1360,6 +1360,10 @@ pool_map_merge(struct pool_map *map, uint32_t version,
 	}
 	D_ASSERT(addr - (void *)dst_tree <= size);
 	D_DEBUG(DB_TRACE, "Merged all components\n");
+
+	/* Update the total target number for the root */
```

Comment:
This file needs a copyright update.  Please install the githook

Reply:
> This file needs a copyright update. Please install the githook

ah, sure.